core/arch/arm/include/arm.h

```diff
@@ -2,28 +2,6 @@
  * Copyright (c) 2015, Linaro Limited
  * SPDX-License-Identifier: BSD-2-Clause
  * All rights reserved.
- *
```

Comment:
Does this deserves an update of the date 2015-2017?

Reply:
I suppose, but I'd rather not review that in this PR as it's quite massive as it is already.

Reply:
true :)